### ISSUE TITLE
New Category: `Aircraft.Engine.Propeller.` in variable_meta_data

### ISSUE THREAD

**Issue by ehariton** [OpenMDAO/Aviary#229]
### Desired capability or behavior.

We have lots of variables in `Aircraft.Engine` that have `propeller` in the name or are only applicable to propeller designs. We may want to make a new category just to hold those values in `variable_meta_data.py`.

i.e. `Aircraft.Engine.PROPELLER_INTEGRATED_LIFT_COEFFICIENT` -> `Aircraft.Engine.propeller.INTEGRATED_LIFT_COEFFICIENT`

The following names have been identified as only pertaining to propellers:
```
Aircraft.Engine.PROPELLER_INTEGRATED_LIFT_COEFFICIENT
Aircraft.Engine.PROPELLER_ACTIVITY_FACTOR
Aircraft.Engine.NUM_PROPELLER_BLADES
Aircraft.Engine.PROPELLER_DIAMETER
Aircraft.Design.MAX_PROPELLER_TIP_SPEED
Dynamic.Mission.PROPELLER_TIP_SPEED
Aircraft.Engine.HAS_PROPELLERS
Dynamic.Mission.INSTALLATION_LOSS_FACTOR
```

**Comment by jkirk5** [comment]
If we decide we'd rather not introduce a 4-depth hierarchy, just `Aircraft.Propeller.` is an alternative we can consider

**Comment by jkirk5** [comment]
resolved with #519
